components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/model/ActionInvocationSuccessResponse.java

```diff
@@ -32,13 +32,14 @@
 public class ActionInvocationSuccessResponse implements ActionInvocationResponse.APIResponse {
 
     private final ActionInvocationResponse.Status actionStatus;
-
     private final List<PerformableOperation> operations;
+    private final ExtendedData data;
```

Comment:
Shall we rename this to some thing like ResponseData, ResponseContext etc.
I prefer ReponseData given the value is data.